### PR TITLE
Fixed #65 Added Django 1.10 support

### LINKED ISSUE #65: Add Django 1.10 support
> ImportError: cannot import name 'NoArgsCommand'

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [tox]
 # Remember to add to .travis.yml if this is added to.
 envlist =
```
Comment (davidszotten) [inline tox.ini]: might be worth adding 1.10 to the travis config, (as an allowed failure until the final release)

Comment (felixxm) [inline tox.ini]: Done.

File: tox.ini [hunk 2]
```diff
@@ -24,12 +24,13 @@ deps =
     lockfile==0.10.2
     coverage
     mock==1.0.1
-    django14: Django==1.4.22
```
Comment (spookylukey) [inline tox.ini]: We were deliberately using specific version numbers here. Having inexact version numbers could cause a lot of confusion in the case of a regression which could be caused by either a new commit to django-mailer, or a change in the latest version of Django 1.X.Y. It gets worse when a local tox environment might be using an old point release, so the developers won't easily reproduce what happens on Travis or for another user.

Comment (felixxm) [inline tox.ini]: I understand your point of view.

Comment (davidszotten) [inline tox.ini]: what's the process for bumping the point versions? should we consider having the unbounded in the allowed failures section?

File: mailer/management/helpers.py [hunk 3]
```diff
@@ -1,17 +1,16 @@
 from optparse import make_option
-
-from django.core.management.base import NoArgsCommand
+from django.core.management.base import BaseCommand
 
 help_msg = "If 1 don't print messagges, but only errors."
 
 
-if NoArgsCommand.option_list:
+if hasattr(BaseCommand, 'option_list') and BaseCommand.option_list:
```
Comment (davidszotten) [inline mailer/management/helpers.py]: maybe `if getattr(BaseCommand, 'option_list', []):`

**Comment by spookylukey** [comment]
Thanks so much!

**Pull request merged**